Source/IGListAdapter.m

```diff
@@ -577,8 +577,11 @@ - (IGListSectionMap *)sectionMapAdjustForUpdateBlock:(BOOL)adjustForUpdateBlock
     return indexPaths;
 }
 
-- (NSIndexPath *)indexPathForSectionController:(IGListSectionController *)controller index:(NSInteger)index {
-    const NSInteger section = [self.sectionMap sectionForSectionController:controller];
+- (NSIndexPath *)indexPathForSectionController:(IGListSectionController *)controller
+                                         index:(NSInteger)index
+                            usePreviousSection:(BOOL)usePreviousSection {
```

Comment:
I think I'll update this to match the previous comment so they are in sync. Good call outs!